packages/twenty-server/src/modules/workflow/workflow-executor/workspace-services/workflow-executor.workspace-service.ts

```diff
@@ -159,4 +178,26 @@ export class WorkflowExecutorWorkspaceService implements WorkflowExecutor {
       workspaceId,
     );
   }
+
+  private async canBillWorkflow() {
+    if (!this.billingService.isBillingEnabled()) {
+      return { canBillWorkflow: true, billingErrorOutput: {} };
```

Comment:
nitpick: I'm unsure about that. If the billing is not enabled, we can bill the workflow?